README.md

```diff
@@ -30,7 +30,10 @@ Installation
 ------------
 
 `transient` is available on [PyPI](https://pypi.org/project/transient/), so the latest
-release can be installed with `pip install transient`. To install `transient` from
+release can be installed with `pip install transient`. It is also available from
+[GNU Guix](http://guix.gnu.org/), and can be installed with `guix install python-transient`.
+
+To install `transient` from
```

Comment:
Looks like the formatting got a little messed up here, can you fix that?

Reply:
To be clear, you mean the line break on 35 and 36? I'd be happy to remove it if that is the case.

Reply:
Yep, just that line break (but please try to keep things to an approximately 80 character line limit). Also, it's probably worth making this same change to the docs at: https://github.com/ALSchwalm/transient/blob/master/docs/installation.md. That way, if someone is just looking at the readthedocs page, they'll still see the guix option